### ISSUE TITLE
`nx generate @nrwl/angular:application` - Cannot find module '@angular-devkit/architect'

### ISSUE THREAD

**Issue by hadrien-toma** [nrwl/nx#4681]
## Current Behavior
Unability to create an angular application in a new workspace.

## Expected Behavior
Ability to successfully create/run the application.

## Steps to Reproduce

```bash
npx create-nx-workspace@latest workspace --preset=empty --cli=nx --interactive=false --nx-cloud=false
cd workspace
yarn add @nrwl/angular
yarn run nx generate @nrwl/angular:application --name=frontend --style=scss --routing=true
```

### Failure Logs
```
npx: installed 220 in 25.719s
Creating a sandbox with Nx...
added 220 packages from 173 contributors and audited 221 packages in 19.536s

25 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

new workspace --preset=empty --no-interactive --no-nxCloud --appName= --linter=eslint --collection=@nrwl/workspace
CREATE workspace/README.md
CREATE workspace/.editorconfig
CREATE workspace/.gitignore
CREATE workspace/.prettierignore
CREATE workspace/.vscode/extensions.json
CREATE workspace/workspace.json
CREATE workspace/nx.json
CREATE workspace/package.json
CREATE workspace/tools/generators/.gitkeep
CREATE workspace/tools/tsconfig.tools.json
CREATE workspace/tsconfig.base.json
CREATE workspace/.prettierrc
CREATE workspace/apps/.gitkeep
CREATE workspace/libs/.gitkeep
npm notice created a lockfile as package-lock.json. You should commit this file.
added 229 packages from 211 contributors and audited 231 packages in 23.588s

25 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Directory is already under version control. Skipping initialization of git.

———————————————————————————————————————————————


>  NX   NOTE  Nx CLI is not installed globally.

  This means that you might have to use "yarn nx" or "npm nx" to execute commands in the workspace.
  Run "yarn global add nx" or "npm install -g nx" to be able to execute command directly.

yarn add v1.21.1
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
warning @nrwl/angular > @nrwl/cypress > @cypress/webpack-preprocessor > debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
warning @nrwl/angular > @nrwl/cypress > ts-loader > micromatch > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning @nrwl/angular > @nrwl/cypress > ts-loader > micromatch > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
[2/4] Fetching packages...
info fsevents@2.3.1: The platform "linux" is incompatible with this module.
info "fsevents@2.3.1" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "@nrwl/angular > jasmine-marbles@0.6.0" has unmet peer dependency "rxjs@^6.4.0".
warning "@nrwl/angular > @nrwl/cypress@11.2.8" has unmet peer dependency "cypress@>= 3 < 7".
warning "@nrwl/angular > @nrwl/cypress > @cypress/webpack-preprocessor@4.1.5" has unmet peer dependency "webpack@^4.18.1".
warning "@nrwl/angular > @nrwl/cypress > @cypress/webpack-preprocessor > babel-loader@8.2.2" has unmet peer dependency "webpack@>=2".
[4/4] Building fresh packages...
success Saved lockfile.
warning Your current version of Yarn is out of date. The latest version is "1.22.5", while you're on "1.21.1".
info To upgrade, run the following command:
$ sudo apt-get update && sudo apt-get install yarn
success Saved 331 new dependencies.
info Direct dependencies
├─ @nrwl/angular@11.2.8
├─ @nrwl/workspace@11.2.8
├─ @types/node@12.12.38
├─ prettier@2.2.1
├─ ts-node@9.1.1
└─ typescript@4.0.5
info All dependencies
├─ @angular-devkit/schematics@11.0.7
├─ @babel/code-frame@7.12.11
├─ @babel/compat-data@7.12.7
├─ @babel/core@7.12.10
├─ @babel/generator@7.12.11
├─ @babel/helper-builder-binary-assignment-operator-visitor@7.10.4
├─ @babel/helper-compilation-targets@7.12.5
├─ @babel/helper-define-map@7.10.5
├─ @babel/helper-explode-assignable-expression@7.12.1
├─ @babel/helper-get-function-arity@7.12.10
├─ @babel/helper-hoist-variables@7.10.4
├─ @babel/helper-member-expression-to-functions@7.12.7
├─ @babel/helper-plugin-utils@7.10.4
├─ @babel/helper-validator-option@7.12.11
├─ @babel/helper-wrap-function@7.12.3
├─ @babel/helpers@7.12.5
├─ @babel/highlight@7.10.4
├─ @babel/parser@7.12.11
├─ @babel/plugin-proposal-async-generator-functions@7.12.12
├─ @babel/plugin-proposal-class-properties@7.12.1
├─ @babel/plugin-proposal-dynamic-import@7.12.1
├─ @babel/plugin-proposal-export-namespace-from@7.12.1
├─ @babel/plugin-proposal-json-strings@7.12.1
├─ @babel/plugin-proposal-logical-assignment-operators@7.12.1
├─ @babel/plugin-proposal-nullish-coalescing-operator@7.12.1
├─ @babel/plugin-proposal-numeric-separator@7.12.7
├─ @babel/plugin-proposal-object-rest-spread@7.12.1
├─ @babel/plugin-proposal-optional-catch-binding@7.12.1
├─ @babel/plugin-proposal-optional-chaining@7.12.7
├─ @babel/plugin-proposal-private-methods@7.12.1
├─ @babel/plugin-proposal-unicode-property-regex@7.12.1
├─ @babel/plugin-syntax-class-properties@7.12.1
├─ @babel/plugin-syntax-top-level-await@7.12.1
├─ @babel/plugin-transform-arrow-functions@7.12.1
├─ @babel/plugin-transform-async-to-generator@7.12.1
├─ @babel/plugin-transform-block-scoped-functions@7.12.1
├─ @babel/plugin-transform-block-scoping@7.12.12
├─ @babel/plugin-transform-classes@7.12.1
├─ @babel/plugin-transform-computed-properties@7.12.1
├─ @babel/plugin-transform-destructuring@7.12.1
├─ @babel/plugin-transform-dotall-regex@7.12.1
├─ @babel/plugin-transform-duplicate-keys@7.12.1
├─ @babel/plugin-transform-exponentiation-operator@7.12.1
├─ @babel/plugin-transform-for-of@7.12.1
├─ @babel/plugin-transform-function-name@7.12.1
├─ @babel/plugin-transform-literals@7.12.1
├─ @babel/plugin-transform-member-expression-literals@7.12.1
├─ @babel/plugin-transform-modules-amd@7.12.1
├─ @babel/plugin-transform-modules-commonjs@7.12.1
├─ @babel/plugin-transform-modules-systemjs@7.12.1
├─ @babel/plugin-transform-modules-umd@7.12.1
├─ @babel/plugin-transform-named-capturing-groups-regex@7.12.1
├─ @babel/plugin-transform-new-target@7.12.1
├─ @babel/plugin-transform-object-super@7.12.1
├─ @babel/plugin-transform-property-literals@7.12.1
├─ @babel/plugin-transform-regenerator@7.12.1
├─ @babel/plugin-transform-reserved-words@7.12.1
├─ @babel/plugin-transform-shorthand-properties@7.12.1
├─ @babel/plugin-transform-spread@7.12.1
├─ @babel/plugin-transform-sticky-regex@7.12.7
├─ @babel/plugin-transform-template-literals@7.12.1
├─ @babel/plugin-transform-typeof-symbol@7.12.10
├─ @babel/plugin-transform-unicode-escapes@7.12.1
├─ @babel/plugin-transform-unicode-regex@7.12.1
├─ @babel/preset-env@7.12.11
├─ @babel/preset-modules@0.1.4
├─ @babel/runtime@7.12.5
├─ @babel/traverse@7.12.12
├─ @cypress/webpack-preprocessor@4.1.5
├─ @nrwl/angular@11.2.8
├─ @nrwl/cypress@11.2.8
├─ @nrwl/workspace@11.2.8
├─ @schematics/angular@11.0.7
├─ @types/istanbul-lib-coverage@2.0.3
├─ @types/istanbul-lib-report@3.0.0
├─ @types/istanbul-reports@3.0.0
├─ @types/json-schema@7.0.7
├─ @types/json5@0.0.29
├─ @types/node@12.12.38
├─ @types/normalize-package-data@2.4.0
├─ @types/yargs-parser@20.2.0
├─ @types/yargs@15.0.13
├─ ajv-keywords@3.5.2
├─ ajv@6.12.6
├─ ansi-escapes@3.2.0
├─ anymatch@3.1.1
├─ arg@4.1.3
├─ argparse@1.0.10
├─ arr-flatten@1.1.0
├─ assign-symbols@1.0.0
├─ async@0.9.2
├─ atob@2.1.2
├─ axios@0.21.1
├─ babel-code-frame@6.26.0
├─ babel-loader@8.2.2
├─ balanced-match@1.0.0
├─ base@0.11.2
├─ big.js@5.2.2
├─ binary-extensions@2.2.0
├─ bluebird@3.7.1
├─ brace-expansion@1.1.11
├─ braces@3.0.2
├─ browserslist@4.16.3
├─ buffer-from@1.1.1
├─ cache-base@1.0.1
├─ camelcase@5.3.1
├─ caniuse-lite@1.0.30001183
├─ chardet@0.7.0
├─ chokidar@3.5.1
├─ ci-info@2.0.0
├─ class-utils@0.3.6
├─ cli-cursor@2.1.0
├─ cli-spinners@2.5.0
├─ cli-width@2.2.1
├─ cliui@6.0.0
├─ clone@1.0.4
├─ collection-visit@1.0.0
├─ color-convert@1.9.3
├─ color-name@1.1.3
├─ colorette@1.2.1
├─ commondir@1.0.1
├─ concat-map@0.0.1
├─ convert-source-map@1.7.0
├─ copy-descriptor@0.1.1
├─ core-js-compat@3.8.3
├─ core-util-is@1.0.2
├─ cosmiconfig@4.0.0
├─ create-require@1.1.1
├─ cross-spawn@6.0.5
├─ debug@2.6.9
├─ decode-uri-component@0.2.0
├─ defaults@1.0.3
├─ diff@4.0.2
├─ ejs@3.1.5
├─ electron-to-chromium@1.3.650
├─ emoji-regex@8.0.0
├─ emojis-list@3.0.0
├─ errno@0.1.8
├─ es-abstract@1.18.0-next.2
├─ es-to-primitive@1.2.1
├─ escalade@3.1.1
├─ escape-string-regexp@1.0.5
├─ esprima@4.0.1
├─ esutils@2.0.3
├─ expand-brackets@2.1.4
├─ external-editor@3.1.0
├─ extglob@2.0.4
├─ fast-deep-equal@3.1.3
├─ fast-json-stable-stringify@2.1.0
├─ figures@2.0.0
├─ filelist@1.0.1
├─ fill-range@7.0.1
├─ find-cache-dir@3.3.1
├─ find-up@4.1.0
├─ flat@5.0.2
├─ follow-redirects@1.13.2
├─ for-in@1.0.2
├─ fork-ts-checker-webpack-plugin@3.1.1
├─ fs.realpath@1.0.0
├─ gensync@1.0.0-beta.2
├─ get-caller-file@2.0.5
├─ get-value@2.0.6
├─ glob-parent@5.1.1
├─ glob@7.1.4
├─ has-ansi@2.0.0
├─ has-flag@4.0.0
├─ has-value@1.0.0
├─ hosted-git-info@2.8.8
├─ iconv-lite@0.4.24
├─ inflight@1.0.6
├─ inherits@2.0.4
├─ is-accessor-descriptor@1.0.0
├─ is-arrayish@0.2.1
├─ is-binary-path@2.1.0
├─ is-callable@1.2.3
├─ is-ci@2.0.0
├─ is-core-module@2.2.0
├─ is-data-descriptor@1.0.0
├─ is-date-object@1.0.2
├─ is-descriptor@1.0.2
├─ is-directory@0.3.1
├─ is-extglob@2.1.1
├─ is-fullwidth-code-point@3.0.0
├─ is-glob@4.0.1
├─ is-interactive@1.0.0
├─ is-negative-zero@2.0.1
├─ is-plain-object@2.0.4
├─ is-regex@1.1.2
├─ is-symbol@1.0.3
├─ is-windows@1.0.2
├─ is-wsl@1.1.0
├─ isarray@1.0.0
├─ isexe@2.0.0
├─ jake@10.8.2
├─ jasmine-marbles@0.6.0
├─ jest-pnp-resolver@1.2.2
├─ jest-resolve@26.6.2
├─ jest-util@26.6.2
├─ js-tokens@3.0.2
├─ js-yaml@3.14.1
├─ jsesc@2.5.2
├─ json-parse-better-errors@1.0.2
├─ json-parse-even-better-errors@2.3.1
├─ json-schema-traverse@0.4.1
├─ jsonc-parser@2.3.1
├─ jsonfile@4.0.0
├─ kind-of@3.2.2
├─ lines-and-columns@1.1.6
├─ load-json-file@4.0.0
├─ loader-utils@1.4.0
├─ locate-path@5.0.0
├─ log-symbols@4.0.0
├─ magic-string@0.25.7
├─ make-dir@3.1.0
├─ make-error@1.3.6
├─ map-visit@1.0.0
├─ memory-fs@0.5.0
├─ memorystream@0.3.1
├─ microevent.ts@0.1.1
├─ micromatch@3.1.10
├─ mimic-fn@1.2.0
├─ minimist@1.2.5
├─ mixin-deep@1.3.2
├─ ms@2.0.0
├─ mute-stream@0.0.7
├─ nanomatch@1.2.13
├─ nice-try@1.0.5
├─ node-releases@1.1.70
├─ normalize-package-data@2.5.0
├─ normalize-path@3.0.0
├─ npm-run-all@4.1.5
├─ object-copy@0.1.0
├─ object-inspect@1.9.0
├─ object.assign@4.1.2
├─ onetime@2.0.1
├─ opn@5.5.0
├─ ora@5.1.0
├─ os-tmpdir@1.0.2
├─ p-limit@2.3.0
├─ p-locate@4.1.0
├─ p-try@2.2.0
├─ pascalcase@0.1.1
├─ path-exists@4.0.0
├─ path-is-absolute@1.0.1
├─ path-key@2.0.1
├─ path-type@3.0.0
├─ picomatch@2.2.2
├─ pidtree@0.3.1
├─ pkg-dir@4.2.0
├─ posix-character-classes@0.1.1
├─ prettier@2.2.1
├─ process-nextick-args@2.0.1
├─ prr@1.0.1
├─ punycode@2.1.1
├─ read-pkg-up@7.0.1
├─ read-pkg@3.0.0
├─ readable-stream@2.3.7
├─ readdirp@3.5.0
├─ regenerate-unicode-properties@8.2.0
├─ regenerator-runtime@0.13.7
├─ regenerator-transform@0.14.5
├─ regexpu-core@4.7.1
├─ regjsgen@0.5.2
├─ regjsparser@0.6.7
├─ repeat-element@1.1.3
├─ require-directory@2.1.1
├─ require-from-string@2.0.2
├─ require-main-filename@2.0.0
├─ resolve-url@0.2.1
├─ resolve@1.19.0
├─ restore-cursor@2.0.0
├─ ret@0.1.15
├─ run-async@2.4.1
├─ rxjs-for-await@0.0.2
├─ safe-buffer@5.1.2
├─ safer-buffer@2.1.2
├─ schema-utils@2.7.1
├─ set-blocking@2.0.0
├─ set-value@2.0.1
├─ shebang-command@1.2.0
├─ shebang-regex@1.0.0
├─ shell-quote@1.7.2
├─ slash@3.0.0
├─ snapdragon-node@2.1.1
├─ snapdragon-util@3.0.1
├─ source-map-resolve@0.5.3
├─ source-map-support@0.5.19
├─ source-map-url@0.4.1
├─ sourcemap-codec@1.4.8
├─ spdx-correct@3.1.1
├─ spdx-exceptions@2.3.0
├─ split-string@3.1.0
├─ sprintf-js@1.0.3
├─ static-extend@0.1.2
├─ string_decoder@1.1.1
├─ string-width@4.2.0
├─ string.prototype.padend@3.1.1
├─ string.prototype.trimend@1.0.3
├─ string.prototype.trimstart@1.0.3
├─ supports-color@7.2.0
├─ through@2.3.8
├─ to-fast-properties@2.0.0
├─ to-object-path@0.3.0
├─ to-regex-range@5.0.1
├─ tree-kill@1.2.2
├─ ts-loader@5.4.5
├─ ts-node@9.1.1
├─ tsconfig-paths-webpack-plugin@3.2.0
├─ tsconfig-paths@3.9.0
├─ type-fest@0.8.1
├─ typescript@4.0.5
├─ unicode-canonical-property-names-ecmascript@1.0.4
├─ unicode-match-property-ecmascript@1.0.4
├─ unicode-match-property-value-ecmascript@1.2.0
├─ unicode-property-aliases-ecmascript@1.1.0
├─ union-value@1.0.1
├─ universalify@0.1.2
├─ unset-value@1.0.0
├─ uri-js@4.4.1
├─ urix@0.1.0
├─ use@3.1.1
├─ util-deprecate@1.0.2
├─ validate-npm-package-license@3.0.4
├─ wcwidth@1.0.1
├─ webpack-node-externals@1.7.2
├─ which-module@2.0.0
├─ which@1.3.1
├─ worker-rpc@0.1.1
├─ wrap-ansi@6.2.0
├─ y18n@4.0.1
└─ yn@3.1.1
Done in 24.70s.
yarn run v1.21.1
$ nx generate @nrwl/angular:application --name=frontend --style=scss --routing=true
Cannot find module '@angular-devkit/architect'
Require stack:
- /home/hadrien_toma_1/gitlab/bashroom-devs/monorepo/monorepo-1612300076/workspace/node_modules/@nrwl/tao/src/commands/ngcli-adapter.js
- /home/hadrien_toma_1/gitlab/bashroom-devs/monorepo/monorepo-1612300076/workspace/node_modules/@nrwl/tao/src/commands/generate.js
- /home/hadrien_toma_1/gitlab/bashroom-devs/monorepo/monorepo-1612300076/workspace/node_modules/@nrwl/tao/index.js
- /home/hadrien_toma_1/gitlab/bashroom-devs/monorepo/monorepo-1612300076/workspace/node_modules/@nrwl/cli/lib/init-local.js
- /home/hadrien_toma_1/gitlab/bashroom-devs/monorepo/monorepo-1612300076/workspace/node_modules/@nrwl/cli/bin/nx.js
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
yarn run v1.21.1
```

### Environment

- `npx --version`: `6.14.6`

**Comment by hadrien-toma** [comment]
It works when doing `yarn add @nrwl/angular @angular-devkit/architect` instead of simply `yarn add @nrwl/angular`, so I am wondering if the error is expected, does someone know?

**Comment by FrozenPandaz** [comment]
No, this is not expected. Reverted the commit that caused this and published `11.2.9` sorry about that everyone.
